x-pack/plugins/ml/server/lib/alerts/register_anomaly_detection_alert_type.ts

```diff
@@ -48,7 +47,7 @@ export type AnomalyDetectionAlertPayload = {
   job_id: string;
   anomaly_score?: number;
   is_interim?: boolean;
-  anomaly_timestamp?: number;
+  anomaly_timestamp?: string;
```

Comment:
Should we keep it as `number`?

Reply:
Fixed in https://github.com/elastic/kibana/pull/167691/commits/639c294c64fde74cde69be2c27f6488ee7870b1e